11-Backbonejs/events.md

```diff
@@ -0,0 +1,207 @@
+
+We listen for Backbone events in the same way we listen for DOM events: using the `.on()` function. For example, to listen for a `update` event on our `bookList`, we would use the following code.
+
+```javascript
+let updateHandler = (list) => {
+  // do some work, handle the update
+};
+bookList.on('update', updateHandler);
+```
+
+Once this code has ben run, whenever our `bookList` emits a `update` event the `updateHandler()` function will be called. Neat!
+
+Why is this important? **It means we can separate code that updates the model from code that updates the DOM.** In practice, this allows us to simplify and DRY our code that handles DOM events.
+
+TODO DPR: diagram. Something like
+```
+     => DOM event handler (jQuery) =>
+DOM                                    Model / Collection
+     <= Backbone event handler     <=
+```
+### Event Vocabulary
+
+Before we go any further, let's review our vocabulary around event handling. This vocab is general - it applies not just to DOM events and Backbone events in JavaScript, but to event handling in any language.
+
+**Trigger:** When an event happens, we say that the event has been _triggered_.
+- A user _triggers_ a `click` event by clicking a button
+- Our code _triggers_ a `update` event on the `bookList` whenever it adds a book to the list
+
+**Emit:** We might also say that an object _emits_ a `click` event. This usually indicates we're focused more on how our program responds the event than on what caused it.
+- The button _emits_ a `click` event
+- The `bookList` _emits_ a `update` event
+
+**Event Handler:** Code that responds to an event is called an _event handler_ (sometimes _handler_ for short). In JavaScript, this will usually take the form of a callback function.
+- In the above code, `updateHandler()` is an _event handler_
+
+**Listen / Register:** When we associate a handler with an event, we say we are now _listening_ for the event. We also say that we have _registered_ an event handler.
+- In the above code, we are _listening_ for a `update` event
+- We have _registered_ the `updateHandler()` function as an event handler
+
+## Adding Models to a Collection
+
+As a demonstration of this technique, we will add functionality for adding a book to our library. This is what the high-level architecture will look like:
+
+TODO DPR: expanded diagram similar to above but w/ event names
+
+1. A user fills out and submits the "Add a book" form, triggering a `submit` DOM event
+1. Our `submit` event handler will read the form data and use it to add a book to the collection
+1. When the book is added, the collection will emit a `update` event
+1. Our `update` event handler will re-build the table of books in the DOM
+
+We will build these components in reverse order, starting with the handler for the `update` event, and then adding code to read the form and add a model to the collection.
+
+### Building the `update` Event Handler
```

Comment:
This section feels like a wall of text with very little action - consider how we might consolidate this